common/src/main/java/feast/common/logging/config/LoggingProperties.java

```diff
@@ -14,30 +14,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package feast.core.job;
+package feast.common.logging.config;
 
-import feast.core.log.Action;
-import feast.core.model.Job;
-import lombok.Builder;
+import feast.common.logging.entry.AuditLogEntry.FeastInstance;
+import javax.validation.constraints.NotNull;
 import lombok.Getter;
 import lombok.Setter;
 
-/** Task to terminate given {@link Job} by using {@link JobManager} */
 @Getter
 @Setter
-@Builder(setterPrefix = "set")
-public class TerminateJobTask implements JobTask {
-  private Job job;
-  private JobManager jobManager;
+public class LoggingProperties {
+  @NotNull private AuditLogProperties audit;
 
-  @Override
-  public Job call() {
-    JobTask.logAudit(
-        Action.ABORT,
-        job,
-        "Aborting job %s for runner %s",
-        job.getId(),
-        jobManager.getRunnerType().toString());
-    return jobManager.abortJob(job);
+  @Getter
+  @Setter
+  public static class AuditLogProperties {
+    private FeastInstance instance;
+
+    // Whether to enable/disable audit logging entirely.
+    private boolean enabled;
+
+    // Whether to enable/disable message level (ie request/response) audit logging.
+    private boolean messageEnabled;
```

Comment:
Can we call this something more descriptive? `enableMessageLogging`?

Reply:
Ok. updated.